### PR TITLE
Korean translation

### PR DESCRIPTION
**Pull request by Shell4026**
I translated Korean.

### CODE REVIEW

File: src/main/resources/assets/icbmclassic/lang/ko_KR.lang [hunk 1]
```diff
@@ -0,0 +1,577 @@
...
+subtitle.icbmclassic:explosion=폭발
+subtitle.icbmclassic:explosion.fire=화염 폭발
+subtitle.icbmclassic:gas.leak=가스 공격
+subtitle.icbmclassic:machine.hum=기계 윙윙거림
+subtitle.icbmclassic:missile.launch=미사일 발사
+subtitle.icbmclassic:power.down=에너지 빔 종료
+subtitle.icbmclassic:explosion.redmatter=적색 물질 폭발
+
+
+subtitle.icbmclassic:explosion.sonicwave=Sonic Blastsubtitle.icbmclassic:meep=Meep Meepsubtitle.icbmclassic:explosion.sonicwave=Sonic Blastsubtitle.icbmclassic:meep=Meep Meepsubtitle.icbmclassic:missile.launch=Missile Launch
```
Comment (DarkGuardsman) [inline src/main/resources/assets/icbmclassic/lang/ko_KR.lang]: Possible format issue?

**Comment by DarkGuardsman** [comment]
Looks good when run back through google translate. Just seems like there is a few of the subtitles missing at the bottom of the file.

**Comment by Shell4026** [comment]
I added missing parts and made some minor changes to the translation.

**Comment by DarkGuardsman** [comment]
thank you

**Pull request merged**